blueprints/gcve/monitoring/README.md

```diff
@@ -0,0 +1,93 @@
+# Google Cloud VMWare Engine Logging Monitoring Module
+
+The Blueprinte sets up Monitoring and Syslog Logging for Google Cloud VMWare Engine Private Clouds.
+
+Infrastructure monitoring and logging for GCVE is commonly set up using a [standalone Bindplane agent](https://cloud.google.com/vmware-engine/docs/environment/howto-cloud-monitoring-standalone). This module deploys the bindplane agent using a Managed Instance Group to accept metrics and Syslog logs from VMware vCenter and forward them to Cloud Monitoring and Cloud Logging. To forward syslog messages, please refer to the following documentation on [how to configure a private cloud for syslog forwarding](https://cloud.google.com/vmware-engine/docs/environment/howto-forward-syslog).
+
+<p align="center">
+  <img src="gcve-mon-diagram.png" alt="GCVE Logging and Monitoring Blueprint">
+</p>
+
+This module deploys and configures the following resources:
+ * Service Account for Bindplane Agent with permissions to write logs/metrics and access Secret Manager
+ * A firewall rule to allow a healthcheck of port TCP 5142 to check whether the agent runs correctly.
+ * Default Monitoring Dashboards for GCVE
+ * A GCE VM template using Debian 11 for the bindplane agent
+ * A GCE Managed Instance Group for the bindplane agent
+ * Secret Manager secrets:
+    * vCenter Username
+    * vCenter Password
+    * vCenter FQDN
+
+Note that to complete the Monitoring setup you need to configure vCenter to send traffic to the Bindplane agent which listens by default on port TCP 5142.
+
+<!-- BEGIN TOC -->
+- [Basic Monitoring setup with default settings](#basic-monitoring-setup-with-default-settings)
+- [Variables](#variables)
+- [Outputs](#outputs)
+<!-- END TOC -->
+
+## Basic Monitoring setup with default settings
+
+```hcl
+
+module "gcve-monitoring" {
+  source         = ".fabric/blueprints/gcve/monitoring"
+  project_id     = "gcve-mon-project"
+  project_create = {
+    billing_account = "0123AB-ABCDEF-123456"
+    parent          = "folders/1234567890"
+    shared_vpc_host = "abcde-prod-net-spoke-0"
+  }
+
+  vm_mon_name             = "bp-agent"
+  vm_mon_type             = "e2-small"
+  vm_mon_zone             = "europe-west1-b"
+  subnetwork_self_link    = "projects/abcde-prod-net-spoke-0/regions/europe-west1/subnetworks/prod-default-ew1"
+  sa_gcve_monitoring      = "gcve-mon-sa"
+  secret_vsphere_server   = "gcve-mon-vsphere-server"
```

Comment:
bundle these into a `vsphere_config` variable

Reply:
Added a vsphere_secrets variable of type object. I liked this name better, to avoid users accidentially put vSphere credentials in the variables, since only secret names are needed for Secret Manager.